### PR TITLE
fix #77885 asistanaliz.com

### LINKED ISSUE #77885: asistanaliz.com
### Issue URL (Ads)
 [https://asistanaliz.com/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fasistanaliz.com%2F)
### Comment
Username: @RadicalNoise
> 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.31
Browser: | Chrome
Stealth mode options: | Hide your search queries, <br>Remove X-Client-Data header from HTTP requests, <br>Strip URLs from tracking parameters, <br>Self-destructing third-party cookies (2880)
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Security:</b><br/> NoCoin Filter List<br/><br/><b>Language-specific:</b><br/> AdGuard Turkish

### PR DESCRIPTION
**Pull request by wout-codes**
#77885

### CODE REVIEW

File: TurkishFilter/sections/specific.txt [hunk 1]
```diff
@@ -43,6 +43,7 @@ fullhdfilmcibaba.com,720pizlemek.com,1080pfilmizle.net,fullhdfilmcidayi.com,hdfi
 !
 temizsozluk.com.tr##a[rel^="sponsored"]
 temizsozluk.com.tr##section > [class*="mx-auto"][class*="text-center"][class*="text-xs"]
+||wishjus.com/t.js$third-party,domain=asistanaliz.com
```
Comment (Sergey-Lyapin) [inline TurkishFilter/sections/specific.txt:46]: to turkish adservers I suppose
`||wishjus.com^$third-party`

**Review by Sergey-Lyapin — CHANGES_REQUESTED** [review]

File: TurkishFilter/sections/specific.txt (hunk 1 above)
Comment (wout-codes) [reply TurkishFilter/sections/specific.txt:46]: Okay as you wish.

Comment (AdamWr) [reply TurkishFilter/sections/specific.txt]: Or maybe better to Base filter?
I saw the same/similar script on one of the Polish website - https://github.com/AdguardTeam/AdguardFilters/commit/029c755967e87ae671fbe1b9ac1b59f3954860ad#diff-fbf56167647d1d9892a9a9ccdd71a6c88bdf70c0d33822f99e0270e936c4b898R6702

Comment (wout-codes) [reply TurkishFilter/sections/specific.txt]: @AdamWr Okay. Can you please add it directly?

Comment (wout-codes) [reply TurkishFilter/sections/specific.txt]: Thank you.

Comment (AdamWr) [reply TurkishFilter/sections/specific.txt]: Done - https://github.com/AdguardTeam/AdguardFilters/commit/3336fa0cf4e183c8a6030deb967a7bcc29b8927a

**Pull request merged**